The Pump is Doing Its Duty

mjmagee991 opened the pull request:
Changed stop duty cycle from 0 to 10
Set pump object to run stop() during instantiation

mjmagee991 committed:
Changed Pump Implementation Based on Testing

Changed stop duty cycle from 0 to 10
Set pump object to run stop() during instantiation

cbolles reviewed (approved):
Looks good! Just something to look out for, when you update the TMS to use the newest EVT-core, you'll have to tweak the code to use fixed point numeric representation

AarC10 reviewed (approved):
Seems good to me besides the floating point issue Collin mentioned. Good work.

mjmagee991 committed:
Updated Heat Pump Driver to Fixed Point PWM

Adjusted period and duty cycle calculations
Updated documentation to reflect new equations
